htdocs/core/class/extrafields.class.php

```diff
@@ -2157,7 +2157,7 @@ public function getOptionalsFromPost($extrafieldsobjectkey, $keyprefix = '', $ke
 				}
 				elseif (in_array($key_type, array('checkbox', 'chkbxlst')))
 				{
-					if (! GETPOSTISSET($keysuffix."options_".$key.$keyprefix)) continue;	// Value was not provided, we should not set it.
+					// even we receve nothing from this input we need to update it because checkbox send nothing when nothing is selected
```

Comment:
Fuc... checkbox.
So what happen if we have 2 extrafields checkboxes and we update one on the form ? This will also erase the other one ?
Also, on my version 11, the checkbox is more implemented with a html checkbox but with a html select2 component. So old code works. Do you have input checkbox on your side ?